release-doc-in-process/all_vf.TTL

```diff
@@ -346,9 +366,87 @@ vf:fulfills     a owl:ObjectProperty ;
+#        rdfs:range      qudt:unit ;
+#        rdfs:comment    "The unit expected for this resource type as a default." .
+
+vf:definedQuantity a    owl:ObjectProperty ;
+        rdfs:label      "defined quantity" ;
+        rdfs:domain     vf:RecipeFlow ;
+        rdfs:range      qudt:QuantityValue ;
+        rdfs:comment    "The amount and unit defined for this recipe flow." .
+
+# TODO: need to define this, something like currency, inventory, work, service; may be other aspects too.
+#vf:resourceCategory a    owl:ObjectProperty ;
+#        rdfs:label      "category" ;
+#        rdfs:domain     vf:ResourceType ;
+#        rdfs:comment    "Defines the category of behavior expected for the type of resource." .
+
+vf:developedBy a        owl:ObjectProperty ;
+        rdfs:label      "developed by" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      foaf:Agent ;
+        rdfs:comment    "The agent who developed this recipe." .
+
+vf:variantOf a           owl:ObjectProperty ;
+        rdfs:label      "variant of" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      vf:Recipe ;
+        rdfs:comment    "This recipe is a variation of another recipe." .
+
+vf:successorOf a        owl:ObjectProperty ;
+        rdfs:label      "successor of" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      vf:Recipe ;
+        rdfs:comment    "This recipe is an updated version of another recipe." .
+
+vf:affected a           owl:ObjectProperty ;
+        rdfs:label      "affected" ;
+        rdfs:domain     vf:RecipeFlow ;
+        rdfs:range      vf:RecipeResource ;
+        rdfs:comment    "Defines the type of resource this recipe flow affects." .
+
+vf:hasDuration a        owl:ObjectProperty ;
+        rdfs:label      "duration" ;
+        rdfs:domain     vf:RecipeProcess ;
+        rdfs:range      time:TemporalDuration ;
+        rdfs:comment    "The calendar duration defined for this type of process." .
+
+vf:durationMultiplier 
+        a               owl:DataTypeProperty ;
+        rdfs:label      "multiplier" ;
+        rdfs:domain     vf:RecipeProcess ;
+        rdfs:range      ???? ; #TODO
+        rdfs:comment    "The calendar duration defined for this type of process." .
+
+vf:resourceBasedOn
+        a                   owl:ObjectProperty ;
+        rdfs:label          "resource typed as" ;
+        rdfs:domain         [ owl:unionOf (vf:Commitment vf:EconomicResource) ] ;
+        rdfs:range          vf:RecipeResource ;
+        rdfs:comment        "A resource's primary classification, part of describing that resource." .
 
-# something for models for GTIN, etc. identifiers
+vf:processBasedOn
+        a                   owl:ObjectProperty ;
+        rdfs:label          "process typed as" ;
+        rdfs:domain         vf:Process ;
+        rdfs:range          vf:RecipeProcess ;
+        rdfs:comment        "A process's primary classification, as part of describing that process." .
+
+vf:deliverable
```

Comment:
This needs to match what we do with Plan. To get started we can simply use `vf:outputOf`

Reply:
Ouch, that is another one of those convoluted range and domain definitions.  Plan does not have its own outputOf though.  Although it could in the sense of defined deliverables which may get changed based on the lower level processes in the plan, I was thinking of adding deliverable there, but am still thinking about differences between plans and operational processes.

I guess in any case it could be vf:outpufOf.  I just hate to add it to that already confusing and fuzzily defined property that has to be clarified in text somewhere that people may or may not see. I wonder if there is some way in rdf to make that more precise and not have to use different vocabulary? :pray: